coderd/provisionerdserver/provisionerdserver_test.go

```diff
@@ -117,6 +118,7 @@ func TestHeartbeat(t *testing.T) {
 	//nolint:dogsled // ｡:ﾟ૮ ˶ˆ ﻌ ˆ˶ ა ﾟ:｡
 	_, _, _, _ = setup(t, false, &overrides{
 		ctx:               ctx,
+		heartbeatDone:     heartbeatDone,
 		heartbeatFn:       heartbeatFn,
 		heartbeatInterval: testutil.IntervalFast,
```

Comment:
Aside: a good example of a test that would be dramatically simpler with the time test library we've talked about.